cmd/logout.js

```diff
@@ -14,12 +14,11 @@
  */
 
 const async = require('async');
-const program = require('commander');
 const project = require('../lib/project.js');
 const user = require('../lib/user.js');
 const handleActionFailure = require('./../lib/util').handleCommandFailure;
 
-function logout(command, cb) {
+function logout(argv, cb) {
   return async.series([
```

Comment:
Is `init()` intentionally omitted from this command?

Reply:
IMO it shouldn't be but it was like this. I intend to change it during the refactoring, though.